### PR TITLE
doc: added contributors graph 

### LINKED ISSUE #56: doc(readme): add contributors graph
## Description
I want to add contributors graph in readme

## Screenshots
Put any screenshot(s) here.

## Are you raising this PR under GSSoC'23? 
Yes

## How are you planning to resolve on this issue?
Introducing few code changes in readme file

### PR DESCRIPTION
**Pull request by S-ishita**
## Related Issue

Closes: #56 

### Describe the changes you've made
few code changes are done in readme file

### Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] Does this code have any breaking changes?
- [x] Does this code have any yaml schema changes?

### Screenshots
Screenshot from 2023-05-21 22-59-23

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,3 +51,9 @@ Links:
 3. https://confidentialcomputing.io
 4. https://differentialprivacy.org 
 
+## Contributors
```
Comment (qascade) [inline README.md:54]: Looks Cool. Would you like to do some more changes or should I close this PR?

Comment (S-ishita) [reply README.md:54]: you can merge it :)

**Review by qascade — APPROVED** [review]
Thanks for the PR. LGTM.

**Comment by gurjeetsinghvirdee** [comment]
Hey @qascade in some PRs you added wrong labels please fix that
these are the correct labels
image

**Comment by qascade** [comment]
@gurjeetsinghvirdee this should solve the issue.

**Pull request merged**